ext/extconf.rb

```diff
@@ -1,5 +1,36 @@
 require 'mkmf'
 
+LIBDIR      = RbConfig::CONFIG['libdir']
+INCLUDEDIR  = RbConfig::CONFIG['includedir']
+
+HEADER_DIRS = [
+  '/opt/local/include',
+  '/usr/local/include',
+  INCLUDEDIR,
+  '/usr/include',
+]
+
+LIB_DIRS = [
+  '/opt/local/lib',
+  '/usr/local/lib',
+  LIBDIR,
+  '/usr/lib',
+]
+
+dir_config('replication', HEADER_DIRS, LIB_DIRS)
```

Comment:
Just `dir_config('replication')` works without `HEADER_DIRS` and `LIB_DIRS`. Can you delete the above changes and put `dir_config('replication')` ?

With `dir_config` command, extconf accepts the `--with-replication-lib` and `--with-replication-include` options.